interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,212 @@
+
+There are many ways to leverage AzureML. Currently DeepSeismic has integrated with AzureML to train a pipeline, which will include creating an experiment titled "DEV-train-pipeline" which will contain all training runs, associated logs, and the ability to navigate seemlessly through this information. AzureML will take data from a blob storage account and the associated models will be saved to this account upon completeion of the run.
+
+Please refer to microsoft docs for additional information on AzureML pipelines and related capabilities https://docs.microsoft.com/en-us/azure/machine-learning/concept-ml-pipelines
+
+## Files needed for this AzureML run
+
+You will need the following files to complete an run in AzureML
+
+- [.azureml/config.json](../../../.azureml.example/config.json) This is used to import your subscription, resource group, and AzureML workspace
+- [.env](../../../.env.example) This is used to import your environment variables including blob storage information and AzureML compute cluster specs
+- [kickoff_train_pipeline.py](dev/kickoff_train_pipeline.py) This script shows how to run an AzureML train pipeline 
+- [cancel_run.py](dev/cancel_run.py) This script is used to cancel an AzureML train pipeline run
+- [base_pipeline.py](base_pipeline.py) This script is used as a base class and kickoff_train_pipeline.py inherits from it. This is intended to be a helpful abstraction that an an future addition of an inference pipeline can leverage
+- [train_pipeline.py](train_pipeline.py) This script inherts from base_pipeline.py and is used to construct the pipeline and its steps. The script kickoff_train_pipeline.py will call the function defined here and the pipeline_config
+- [pipeline_config.json](pipeline_config.json) This config specifies the steps of the pipeline, location of data, and any specific arguments. This is consumed once the kickoff_train_script.py is run
+- [train.py](../../../experiments/interpretation/dutchf3_patch/local/train.py) This is the training script that is used to train the model
+- [unet.yaml](../../../experiments/interpretation/dutchf3_patch/local/configs/unet.yaml) This config specifices the model configuration to be used in train.py and is referenced in the pipeline_config.json
+- [azureml_requirements.txt](../../../experiments/interpretation/dutchf3_patch/local/azureml_requirements.txt) This file holds all dependencies for train.py so they can be installed on the compute in Azure ML
+- [logging.config](../../../experiments/interpretation/dutchf3_patch/local/logging.config) This config is used to set up logging
+- local environment with cv_lib and interpretation set up using guidance [here](../../../README.md)
+
+## Running a Pipeline in AzureML
+Set the following environment variables:
+```
+BLOB_ACCOUNT_NAME
+BLOB_CONTAINER_NAME
+BLOB_ACCOUNT_KEY
+BLOB_SUB_ID
+AML_COMPUTE_CLUSTER_NAME
+AML_COMPUTE_CLUSTER_MIN_NODES
+AML_COMPUTE_CLUSTER_MAX_NODES
```

Comment:
Specify that this sets the max number of nodes in the cluster, when submitting multiple jobs. This is _not_ the max number of nodes for multi-node training for a single job

Reply:
Added clarity in this commit: https://github.com/microsoft/seismic-deeplearning/pull/195/commits/f2269c517f6e25c06999ac06be228af73e50158f